src/Task01_FreeCurrencyAPI/currencyApi.js

```diff
@@ -0,0 +1,64 @@
+export const apiKey = process.env.API_KEY
+
+export async function fetchCurrencyDataForDate (date, selectedCurrencies) {
+  const currencies = selectedCurrencies.join(',')
+  const url = `https://api.currencyapi.com/v3/historical?apikey=${apiKey}&currencies=${encodeURIComponent(currencies)}&date=${date}`
```

Comment:
I know it is not secure to commit .env files, but could you please add a template env file without actual api key? so that other developers will know where to put api keys